include/seahorn/MemSimulator.hh

```diff
@@ -31,20 +31,20 @@ class MemSimulator {
   // -- start of internally allocated memory
   unsigned m_intMemStart;
```

Comment:
`MemSimulator` is dead code. If it gets too hard to update it, we can remove it.

Reply:
`HornCex.cc` is the only place that still uses it, what is our plan for that file?

Reply:
We can refactor anything we want. There are no other dependencies other than our code.